openedx/features/pakx/cms/custom_settings/views.py

```diff
@@ -70,6 +82,10 @@ def post(self, request, course_key_string):
         course_experience = request.POST.get('course_experience', 0)
         publisher_logo_url = request.POST['publisher-logo-url']
         publisher_card_logo_url = request.POST['publisher_card_logo_url']
+        days_to_unlock = int(request.POST.get('days-duration') or 0)
```

Comment:
update to `int(request.POST.get('days-duration', 0))`

Reply:
@emzubair I believe the current version one is fine because `get('days-duration') or 0` can handle empty string but `get('days-duration', 0)` cannot.

Reply:
@emzubair let me know